File: Selenium/Project/LMS_Activity4.java
package project;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.openqa.selenium.support.ui.ExpectedConditions;

public class LMS_Activity4 {

	public static void main(String[] args) {

		System.setProperty("webdriver.chrome.driver", "path/to/chromedriver");

		WebDriver driver = new ChromeDriver();

		try {

			driver.get("https://alchemy.hguy.co/lms");

			WebDriverWait wait = new WebDriverWait(driver, Duration.ofMillis(20));
			wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".ld-course-grid")));

			WebElement secondCourseTitle = driver.findElement(By.xpath("(//h3[@class='ld-course-title'])[2]"));

			String courseTitleText = secondCourseTitle.getText();

			if ("Email Marketing Strategies".equals(courseTitleText)) {
				System.out.println("Course title matches successfully.");
			} else {
				System.out.println("Course title does not match. The actual title is: " + courseTitleText);
			}
		} catch (Exception e) {
			e.printStackTrace();
		} finally {

			driver.quit();
		}
	}
}

File: Selenium/Project/lms_Activity4.py
#Get the title of the second most popular course.
#Make sure it matches “Email Marketing Strategies” exactly.

# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

    # Start the Driver
with webdriver.Firefox() as driver:

    # Navigate to the URL
    driver.get("https://alchemy.hguy.co/lms/")
    driver.maximize_window()

    # Get the heading of the webpage.
    course2_Title = driver.find_element(By.XPATH, ".//div[contains(@class,'ld-course-list-items ')]/div[2]/article[1]/div[2]/h3[1]")

    # Verify the heading of the webpage.
    if(course2_Title.text =="Email Marketing Strategies"):
        print ("The title of the second most popular course is : ",course2_Title.text)
    else:
        print ("Wrong Website ") 

    #close the browser
    driver.quit()